daemon/top_unix.go

```diff
@@ -130,6 +142,26 @@ func psPidsArg(pids []uint32) string {
 	return string(b)
 }
 
+// insureOPid add a prefix '-o pid' if there is -o in psArgs
+func insureOPid(psArgs string) (string, bool) {
+	var blHaso = false
+	psArr := fieldsASCII(psArgs)
```

Comment:
Note you do `fieldsASCII(psArgs)` here and also in `ContainerTop()`. Perhaps there is no need to do it twice?

Reply:
No fieldsASCII(psArgs) in ContainerTop